cpp/oneapi/dal/algo/connected_components/common.hpp

```diff
@@ -68,10 +74,15 @@ class descriptor_base : public base {
 
 /// Class for the Connected Components algorithm descriptor
 ///
-/// @tparam Float The data type of the result
-/// @tparam Method The algorithm method
-/// @tparam Task   The task to solve by the algorithm
-/// @tparam Allocator   Custom allocator for all memory management inside the algorithm
+/// @tparam Float       This parameter is not used for Connected Components algortihm.
+///                     The floating-point type that the algorithm uses for
+///                     intermediate computations. Can be :expr:`float` or
+///                     :expr:`double`. 
```

Comment:
I think the 2nd and 3rd sentences should be removed. It is enough to specify that this parameter is not used in the algorithm. Also for Subgraph isomorphism
@outoftardis, what do you think about this?

Reply:
agree